### DISCUSSION TITLE
Version 1.0.0-beta.1

### DISCUSSION THREAD

**Discussion by jbtronics** [Part-DB/Part-DB-server#212] [Announcements]
# Version 1.0.0-beta.1

This is the pre-release version of Part-DB 1.0.0.
Please test it and report bugs in the issues.

No (big) features will be added until the final release, the goal is to find remaining bugs and fix them until then.
Depending on the bugs the final 1.0.0 release will be published in one week (around 12th february 2023). I will try to further improve the documentation until then.

Also feel free to help to improve translations for other languages: https://part-db.crowdin.com/part-db

<hr /><em>This discussion was created from the release <a href='https://github.com/Part-DB/Part-DB-symfony/releases/tag/v1.0.0-beta.1'>Version 1.0.0-beta.1</a>.</em>

**Comment by Pyromane** [comment]
Thank you for the further development!

Although according to crowdin German is fully translated, a few translations seem to be missing here:
![image](https://user-images.githubusercontent.com/5962901/218262096-5cd60ee0-7034-47b1-835d-26e67cd06b3d.png)
